src/renderer/components/hotbar/hotbar-menu.tsx

```diff
@@ -43,7 +44,9 @@ export class HotbarMenu extends React.Component<Props> {
 
   render() {
     const { className } = this.props;
-    const hotbarIndex = HotbarStore.getInstance().activeHotbarIndex + 1;
+    const hotbarStore = HotbarStore.getInstance();
+    const hotbar = hotbarStore.getActive();
+    const hotbarIndex = hotbarStore.activeHotbarIndex + 1;
```

Comment:
This line is confusing, maybe rename to `activeIndexDisplay`?

Reply:
Done.